x-pack/plugins/secops/public/apps/testing_app.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { compose } from '../lib/compose/testing_compose';
+import { startApp } from './start_app';
+startApp(compose());
```

Comment:
nit: I'd have one additional space to separate the code from the import above it to be consistent with all the other great code you have here.

Reply:
`with all the other great code you have here` lol

Reply:
trying to make me feel better ;)